### PR TITLE
Allow triggering of ingest lambda direct from SQS.

### PR DESCRIPTION
**Pull request by movestill**
Ugly code, but didn't want to do a full refactor.

This will let us connect a large ingest queue directly to the tile ingest lambda so AWS can manage draining of the queue.

Not tested yet.

### CODE REVIEW

File: lambda/tile_ingest_lambda.py [hunk 1]
```diff
@@ -87,8 +105,10 @@ def handler(event, context):
     tile_index_result = tile_index_db.getCuboid(msg_data["chunk_key"], int(msg_data["ingest_job"]))
     if tile_index_result is None:
         # If chunk_key is gone, another lambda uploaded the cuboids and deleted the chunk_key afterwards.
-        # Remove message so it's not redelivered.
-        ingest_queue.deleteMessage(msg_id, msg_rx_handle)
+        if not sqs_triggered:
```
Comment (sandyhider) [inline lambda/tile_ingest_lambda.py]: Why is it only deleting message when not sqs_triggered, does the message automatically get pulled from the queue and added to an async lambda call?

Comment (movestill) [reply lambda/tile_ingest_lambda.py]: When triggering via SQS, AWS handles message deletion for you.

File: lambda/tile_ingest_lambda.py [hunk 2]
```diff
@@ -41,39 +42,56 @@ def handler(event, context):
...
-            msg_data = json.loads(msg[2])
-            print("MESSAGE DATA: {}".format(msg_data))
-            break
-        else:
-            rx_cnt += 1
-            print("No message found. Try {} of 6".format(rx_cnt))
-            time.sleep(1)
+    # Used as a guard against trying to delete the SQS message when lambda is
+    # triggered by SQS.
+    sqs_triggered = False
```
Comment (sandyhider) [inline lambda/tile_ingest_lambda.py]: should the boolean be this?
sqs_triggered = 'Records' in event and len(event['Records']) > 0

**Comment by sandyhider** [comment]
Merging to master and production-endpoint.

**Pull request merged**